zephyr/module.yml

```diff
@@ -4,18 +4,18 @@ build:
   kconfig: Kconfig
 blobs:
   - path: liblt_9518_zephyr.a
-    sha256: f50739f06bf78be1cdafac310b5d923e59517e658d9209d989a06feaaed0847a
+    sha256: 354B2F972F4A9012A66C2015A7092591EEC18F29A06BA8F50EC25BD0D3CF9A31
     type: lib
-    version: 'V4.0.1.1_patch_001'
+    version: 'V4.0.1.1_patch_002'
     license-path: tlsr9/ble/license.txt
     url: https://github.com/telink-semi/zephyr_hal_telink_b91_ble_lib/raw/e1fa1784dc4b144b4c033ecbe236f3f31099439e/liblt_9518_zephyr.a
```

Comment:
I think due to new revision this paths should be updated...

Reply:
Done